components/director/internal/open_resource_discovery/fixtures_test.go

```diff
@@ -608,6 +641,44 @@ func fixBundleCreateInput() []*model.BundleCreateInput {
 	}
 }
 
+func fixAPIsWithLowerVersion() []*model.APIDefinition {
+	apis := fixAPIs()
+
+	for _, api := range apis {
+		api.Version.Value = "1.1.0"
```

Comment:
From my POV this is not necessary because:
    - if we do not lower the version here, the hash below will be with the default version that is defined in the fixture itself; that will be our `*fromDb resource`;
    - in the `resourceHashes` map which represents the `input` resources, the hash will again include the version value defined in the fixture;
    - so, hashes for both `input` and `*fromDB` resources will have the same version value -> then in the test cases where some field will be changed without version bump, the new hash will be re-evaluated and it will see that there are changes but the versions are still equal -> validation error

To, sum it up, we will simplify things by not having to explicitly lower the version field in the test cases -> `doc.<Resource>[0].Version = fixPackagesWithLowerVersion()[0].Version` will be obsolete.